### PR TITLE
Fix SwiftLint violation in fonts template

### PR DESCRIPTION
**Pull request by ryanmasondavies**
* [x] I've added an entry in the `CHANGELOG.md` file to explain my changes and credit myself
  * [x] Add the entry in the appropriate section (Bug Fix / New Feature / Breaking Change / Internal Change)
  * [x] Add a period and 2 spaces at the end of your short entry description
  * [x] Add links to your GitHub profile to credit yourself and to the related PR and issue after your description

---

As per #436.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -21,6 +21,10 @@
   [Darron Schall](https://github.com/darronschall)
   [#463](https://github.com/SwiftGen/SwiftGen/issues/463)
   [#464](https://github.com/SwiftGen/SwiftGen/pull/464)
+* Fonts: disable a warning in generated font files for projects with `conditional_returns_on_newlines` SwiftLint rule enabled.
```
Comment (AliSoftware) [inline CHANGELOG.md]: Please add 2 spaces at the end of that line (see other entries for inspiration). _That's a formatting trick in Markdown to make the next line be on its own line once rendered_

File: templates/fonts/swift3.stencil [hunk 2]
```diff
@@ -23,7 +23,9 @@
   }
 
   {{accessModifier}} func register() {
+    // swiftlint:disable conditional_returns_on_newline
     guard let url = url else { return }
+    // swiftlint:enable conditional_returns_on_newline
```
Comment (AliSoftware) [inline templates/fonts/swift3.stencil]: You could also instead directly use `// swiftlint:disable:this conditional_returns_on_newline` just on the affected line,
or `// swiftlint:disable:next conditional_returns_on_newline` on the previous line. This avoids the need to re-enable it on the next line like here.

File: templates/fonts/swift4.stencil [hunk 3]
```diff
@@ -23,7 +23,9 @@
   }
 
   {{accessModifier}} func register() {
+    // swiftlint:disable conditional_returns_on_newline
     guard let url = url else { return }
+    // swiftlint:enable conditional_returns_on_newline
```
Comment (AliSoftware) [inline templates/fonts/swift4.stencil]: same

**Review by AliSoftware — COMMENTED** [review]
Thanks for the PR!

Just some nitpicks to address.

**Comment by AliSoftware** [comment]
Also, since you changed the templates, you'll need to fix the unit tests — as they're now generating a different output than what's expected by the tests.

To do that, you could either modify the files in `fixtures/generated/` manually to match your changes in the templates… or use the "Templates - Generate Output" scheme that will automate the re-generation of those (but in that latter solution, be sure to then inspect the git diff to manually check that the changes applied to the generated files match what you expect)

File: templates/fonts/swift3.stencil (hunk 2 above)
Comment (iotize) [reply templates/fonts/swift3.stencil]: TIL! This is much neater.

**Comment by iotize** [comment]
Thanks for the feedback! The issues should be resolved now.

**Review by AliSoftware — APPROVED** [review]

**Pull request merged**